docs/book/en/src/cli/wasmedge.md

```diff
@@ -41,7 +41,8 @@ The options of the `wasmedge` CLI tool are as follows.
     * WasmEdge will execute the function which name should be given in `ARG[0]`.
     * If there's exported function which names `_initialize`, the function will be executed with the empty parameter at first.
 4. (Optional) `--dir`: Bind directories into WASI virtual filesystem.
-    * Use `--dir guest_path:host_path` to bind the host path into the guest path in WASI virtual system.
+    * Use `--dir host_path` to bind the host path in WASI virtual system.
```

Comment:
We should show the equal mapping with `:` format.

Reply:
Done